app/validators/answers/option_answer_validator.py

```diff
@@ -0,0 +1,79 @@
+from functools import cached_property
+
+from app.validators.answers import AnswerValidator
+
+
+class OptionAnswerValidator(AnswerValidator):
+    DUPLICATE_LABEL_FOUND = "Duplicate label found"
+    DUPLICATE_VALUE_FOUND = "Duplicate value found"
```

Comment:
I'm going to disagree on these - they're framed as events to hint at the fact they're errors rather than static vars. I'll change these to prefix with found.

Reply:
Actually, my use of these seems inconsistent so I've dropped it.